### ISSUE TITLE
[Android][0.60.0] AccessibilityLabels has extra characters coma and space at the end

### ISSUE THREAD

**Issue by dsramier** [facebook/react-native#25581]
React Native version: 0.60

## Steps To Reproduce

1. add an accessibilityLabel to an element

2. build your app

3. use a tool (we are using Appium) to read/interact with your accessibilityID on your Android app

4. your accessibility ID presents an additional coma and space 

Describe what you expected to happen:
Expect same behavior than .59 version
to have accessibilityID = TestLabel

Snack, code example, or link to a repository:
<Button
              accessibilityLabel={'TestLabel'}
              onPress={() => this.onPressLogin()}
              hierarchy={ButtonHierarchy.Secondary}
            />

**Comment by cassioseffrin** [comment]
Any update ? I am having the same issue. In my projects I used AccessibilityLabel for automated tests. After update all tests get failure.

**Comment by dsramier** [comment]
Hello,

Unfortunately still didn't get any support :(

Regards

**Comment by wswebcreation** [comment]
It doesn't matter if you inspect the element with Appium or just with the Android Studio Layout inspector (because Appium translates the the views to XML), the `, ` is being added to each `accessibilityLabel`, see the image below.

I've also been trying to find the code in RN that is responsible for the translation in Android to translate the `accessibilityLabel` to `content-desc`, but I couldn't find it. If someone can point me to that I can try to fix it with a PR

![image](https://user-images.githubusercontent.com/11979740/62421452-492aa700-b6a2-11e9-85c1-e3507f1b88c8.png)

**Comment by FutureGUIs** [comment]
It looks like this is the suspect PR that broke things for us:  [https://github.com/facebook/react-native/commit/1aeac1c](url)  

And this line where it's appending the comma even though it may not be necessary: [https://github.com/facebook/react-native/blob/1aeac1c62528004d994200664368dc85fba1795d/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L163](url)

**Comment by wswebcreation** [comment]
@FutureGUIs 

Thanks for finding this, I made a remark in the PR and asked for the background of this change

**Comment by Asvarox** [comment]
Seems it's been fixed with https://github.com/facebook/react-native/commit/812abfd and the fix was released with `0.60.5`. I think it might could be closed now?

**Comment by dsramier** [comment]
I just give a try and I still have that coma after the accessibility label

**Comment by maestor** [comment]
0.60.5 fixed issue for me.

**Comment by dsramier** [comment]
Same on my side It is working again. Thanks for that!!!

**Comment by wswebcreation** [comment]
Updated my app to `0.60.5` and it works, I think this issue can be closed now

**Comment by cassioseffrin** [comment]
git add .
git commit -m 'update v0.60'
react-native upgrade

These steps above have solved the issue, Thks!
